node/forest_libp2p/src/behaviour.rs

```diff
@@ -55,6 +63,7 @@ impl NetworkBehaviourEventProcess<MdnsEvent> for ForestBehaviour {
             MdnsEvent::Discovered(list) => {
                 for (peer, _) in list {
                     trace!("mdns: Discovered peer {}", peer.to_base58());
+                    self.connect(peer.clone());
```

Comment:
should this also be called on manual dials? I can't comment anywhere else (and isn't really a thing rn) but we probably will introduce functionality to dial peers manually through cli or otherwise and this would be forgotten by then

Reply:
good catch